### DISCUSSION TITLE
Issue with `~` Input in Ghostty Terminal

### DISCUSSION THREAD

**Discussion by maluta** [ghostty-org/ghostty#4190] [Q&A]
**Environment:**
- **Application**: Ghostty Terminal  
- **Version**: 1.0.1-main+ecfca17a (built from source `zig build -Doptimize=ReleaseFast`)  
- **Operating System**: Ubuntu 24.10  
- **Window Manager**: i3wm (also reproduced on Ubuntu Wayland)

**Issue Description:**
When attempting to use the `~` character in Ghostty Terminal, it does not function as expected. (Refer to the attached animated GIF for demonstration.)  

**Steps to Reproduce:**
1. Open Ghostty Terminal.
2. Attempt to input the `~` character.
3. Observe the behavior.

**Expected Behavior:**  
The `~` character should display and behave as normal without causing issues.

**Actual Behavior:**  
The input of the `~` character does not work correctly. The issue persists across both i3wm and Ubuntu Wayland environments.

**Attachments:**  
- Animated GIF demonstrating the issue.

![Peek 31-12-2024 10-47](https://github.com/user-attachments/assets/d5afcc7c-f9f0-43df-aba5-ba9cee2139f3)

**Comment by rhodes-b** [comment]
I dont think this is an issue.

Can you try
`shell-integration = none` in your config and see if that behavior happens
